traces/traces_test.go

```diff
@@ -0,0 +1,27 @@
+package traces
+
+import (
+	"context"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTraceInit(t *testing.T) {
+	t.Parallel()
```

Comment:
`t.Parallel()` can have an effect without `t.Run()` -- by default, all tests within a package run sequentially rather than in parallel, and the call to `t.Parallel()` here would change that behavior if/when any other test cases are added to this package.

It doesn't look like we're consistent with `t.Parallel()` in this repo -- [some tests](https://github.com/osquery/osquery-go/blob/master/client_test.go) use it, [others](https://github.com/osquery/osquery-go/blob/master/plugin/config/config_test.go) don't. I like it as a default, so I would be inclined to keep it here unless you feel strongly about it?